README.md

```diff
@@ -60,6 +60,19 @@ Twig components can also be generated by changing directory into the project:
 yo hozokit:component
 ```
 
+#### Further Steps
+
+Once the generator has finished running the following steps might be required if not done already:
+
+1. Setup a webserver cabable of running php and create a mySQL database for Wordpress. Read [how to install Wordpress](https://wordpress.org/support/article/how-to-install-wordpress/) to learn more.
+
+1. Install Hozokit Node dependencies for your theme. (required if the installing dependencies step has not been sucessful)

```

Comment:
To avoid confusion, it may be clearer to state it like this..

If dependencies fail to install: manually install Hozokit Node dependencies for your theme.

Reply:
Changed step to match to this text. Thanks @arrontp